tests/Behat/Context/Cli/InstallingPluginContext.php

```diff
@@ -105,10 +111,12 @@ public function roleShouldHaveAccessToEverySection(string $roleName): void
             return $permission->type();
         }, $administrationRole->getPermissions());
 
-        $availableSyliusSections = $this->syliusSectionsProvider->__invoke();
+        $availablePermissions = array_map(function (string $section): string {
+            return $this->sectionsToPermissionsMapper->map($section);
+        }, $this->syliusSectionsProvider->__invoke());
```

Comment:
We could extract this to the private function (`provideAndMapPermissions`?) to make the main function more readable